ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -33,7 +32,7 @@
           </td>
         </tr>
 
-        <tr v-else :key="row.item.id" class="owner-info">
+        <tr v-else-if="row.item.id !== undefined" :key="row.item.id" class="owner-info">
```

Comment:
Could this be simplified to `row.item.id` or `!!row.item.id` ?

Reply:
Good catch!